### DISCUSSION TITLE
Crash on iOS 12.2

### DISCUSSION THREAD

**Discussion by fzn0268** [rustdesk/rustdesk#9029] [General]
### Bug Description

The latest release(1.2.7) crashes when opening app on iOS 12.2(iPhone Xs Max).

### How to Reproduce

1. Install latest release(1.2.7) on App Store
2. Tap "Open" button on App Store or app icon on home screen

### Expected Behavior

It can open and use normally.

### Operating system(s) on local side and remote side

iOS 12.2 -> Windows 10 or macOS 12

### RustDesk Version(s) on local side and remote side

1.2.7 -> 1.2.7

### Screenshots

https://github.com/user-attachments/assets/c9f0fc12-c885-4b85-837b-3d26e9a7f27f

**Comment by rustdesk** [comment]
Try out uninstall and install again. Or you need to upgrade iOS.

**Reply by fzn0268** [reply]
Tried reinstall, no luck. Before update, the previous version(1.2.2) was normal. Is there any change not compatible with iOS 12 in the current release? The system requirement is still iOS 12.0 on app store page now.
